# sys

int8 planning_change_Rsp_status 

int8 planning_change_Rsp_response

float32 planning_change_lat_error

int8 PlanningTraj_IsValid

float32 coeffs_0

float32 coeffs_1

float32 coeffs_2

float32 coeffs_3

float32 coeffs_4

float32 coeffs_5
